SimCalorimetry/EcalElectronicsEmulation/src/EcalSimRawData.cc

```diff
@@ -63,44 +57,50 @@ EcalSimRawData::EcalSimRawData(const edm::ParameterSet &params) {
 
   iEvent = 0;
 
-  string writeMode = params.getParameter<string>("writeMode");
+  std::string writeMode = params.getParameter<std::string>("writeMode");
 
-  if (writeMode == string("littleEndian")) {
+  if (writeMode == std::string("littleEndian")) {
     writeMode_ = littleEndian;
-  } else if (writeMode == string("bigEndian")) {
+  } else if (writeMode == std::string("bigEndian")) {
     writeMode_ = bigEndian;
   } else {
     writeMode_ = ascii;
   }
+
+  eeSrFlagToken_ = consumes<EESrFlagCollection>(edm::InputTag(srDigiProducer_, eeSrFlagCollection_));
+  ebSrFlagToken_ = consumes<EBSrFlagCollection>(edm::InputTag(srDigiProducer_, ebSrFlagCollection_));
+  ebDigisToken_ = consumes<EBDigiCollection>(edm::InputTag(digiProducer_, ebDigiCollection_));
+  trigPrimDigisToken_[0] = consumes<EcalTrigPrimDigiCollection>(edm::InputTag(tpProducer_, tcpDigiCollection_));
+  trigPrimDigisToken_[1] = consumes<EcalTrigPrimDigiCollection>(edm::InputTag(tpProducer_, tpDigiCollection_));
 }
 
 void EcalSimRawData::analyze(const edm::Event &event, const edm::EventSetup &es) {
   // Event counter:
   ++iEvent;
 
   if (xtalVerbose_ | tpVerbose_) {
-    cout << "=================================================================="
-            "====\n"
-         << " Event " << iEvent << "\n"
-         << "------------------------------------------------------------------"
-            "----\n";
+    std::cout << "=================================================================="
```

Comment:
@bsunanda , why std::cout and no end of line? I mean "/n" as a last symbol not always complete the line.